src/components/StatusBar/styles/StatusBar.css.js

```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components'
+import forEach from '../../../styles/utilities/forEach'
+import { getColor, rgba } from '../../../styles/utilities/color'
+import Centralize from '../../Centralize'
+
+const statusStyles = {
+  error: 'red',
+  info: 'blue',
+  success: 'green',
+  warning: 'yellow',
+}
+
+const statusBarStyleBold = color =>
+  `
+    background-color: ${getColor(`${color}.500`)};
+    border-bottom: 1px solid ${getColor(`${color}.600`)};
+    color: white;
+    font-weight: bold;
+    .c-StatusBarButton{
+        background-color: ${getColor(`${color}.700`)};
+        text-shadow: 0 1px 1px ${rgba('black', 0.1)};
+    }
+    `
+
+const statusBarStyleLight = color =>
+  `
+    background-color: ${getColor(`${color}.200`)};
+    border-bottom: 1px solid ${getColor(`${color}.300`)};
+    color: ${getColor(`${color}.500`)};
+
+    .c-StatusBarButton{
+        background-color: ${getColor(`${color}.500`)};
+        &:focus {
+        box-shadow: 0 0 0 2px ${rgba('white', 0.8)};
+        }
+    }
+    `
+
+const makeStatusStyles = () => {
+  return forEach(
+    statusStyles,
+    (name, value) => `
```

Comment:
The same question here as above.

Reply:
I did update the code to do an explicit return here 0c7afaa5